context/app/static/js/shared-styles/chips/SelectableChip/SelectableChip.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Chip, { ChipProps } from '@mui/material/Chip';
+import CheckRoundedIcon from '@mui/icons-material/CheckRounded';
+
+interface SelectableChipProps extends ChipProps {
+  isSelected: boolean;
+}
+
+function SelectableChip({ isSelected, ...rest }: SelectableChipProps) {
```

Comment:
There is a component in `FileBrowser` called `ControlChip` which is very similar + contains a minor styling change to make the width of the chip consistent when selected and deselected to avoid layout shifts; can we merge that functionality into this? I should have split that out into a separate file when I did that conversion.